docs/user-guide/gpus.md

```diff
@@ -0,0 +1,136 @@
+---
+assignees:
+- vishh
+title: GPU Support
+---
+
+Kubernetes includes **experimental** support for managing NVIDIA GPUs spread across nodes.
+This page describes how users can consume GPUs and the current limitations.
+
+## Pre-requisites
+
+1. Kubernetes nodes have to be pre-installed with Nvidia drivers. Kubelet will not detect Nvidia GPUs otherwise.
```

Comment:
Assume I have a machine and I don't know if the drivers are installed.  How do I check?  (Update the doc)